packages/noya-renderer/src/components/EditablePath.tsx

```diff
@@ -104,6 +148,36 @@ export default function EditablePath({
           />
         );
       })}
+
+      {parsedCurvePoints.map((points, index) => {
```

Comment:
We're currently filtering `parsedCurvePoints` for `curveMode === 1` twice. Let's do a `.filter` before the `return` so we can share this logic

E.g. `const controlPoints = parsedCurvePoints.filter(...)`

Reply:
Within these `.map`, we should rename `points` => `point`, since plural implies an array

Reply:
Done.